compiler_gym/bin/service.py

```diff
@@ -257,9 +257,7 @@ def print_service_capabilities(env: CompilerEnv):
             )
             print(table)
         else:
-            raise NotImplementedError(
-                "Only Commandline and NamedDiscrete are supported."
-            )
+            print(f"{type(action_space).__name__} not supported.")
```

Comment:
How about just printing `print(action_space)` and relying on whatever `__repr__` it defines?

Reply:
Done.